README.md

```diff
@@ -47,12 +47,16 @@ Coming soon! Readings should appear at least 1 month before the date of discussi
 
 ### Alternatives to Capitalist Structures
 
-Topic description coming soon!
+When we explore current centralized data models, many of our fears and challenges are driven by the power of capitalist incentives: reduction of privacy, control from advertisers, and concentration of data ownership into a few corporations are all justified by capitalism's imperative for delivery of maximum value to shareholders.
```

Comment:
```suggestion
When we explore current centralized data models, many of our fears and challenges are driven by the power of capitalist incentives; the reduction of privacy, disproportional influence by advertisers, and concentrated ownership of data by a few corporations are all seemingly justified by the capitalist imperative to deliver maximum value to shareholders.
```

Reply:
@Frijol suggested above

Reply:
thanks! (Also, cool feature! I didn't know you could do that)